### PR TITLE
Link to embassy-usb

### PR DESCRIPTION
**Pull request by stappersg**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,3 +34,10 @@ Class crates
 
 * [usbd-serial](https://github.com/mvirkkunen/usbd-serial) [![Crates.io](https://img.shields.io/crates/v/usbd-serial.svg)](https://crates.io/crates/usbd-serial) - CDC-ACM serial port class
 * [usbd-hid](https://github.com/twitchyliquid64/usbd-hid) [![Crates.io](https://img.shields.io/crates/v/usbd-hid.svg)](https://crates.io/crates/usbd-hid) - HID class
+
+
+Others
+------
+
+Other implementations for USB in Rust
+* [embassy-usb](https://github.com/embassy-rs/embassy/blob/master/embassy-usb/src/driver.rs)
```
Comment (ryan-summers) [inline README.md:43]: It would be good to explicitly note that embassy-usb is an async variant. Async isn't widely available on stable Rust at the moment due to GATs.

**Review by ryan-summers — CHANGES_REQUESTED** [review]

**Comment by stappersg** [comment]
On Sun, May 22, 2022 at 02:10:51AM -0700, Ryan Summers wrote:
> It would be good to explicitly note that embassy-usb is an async variant.

Acknowledge


> Async isn't widely available on stable Rust at the moment due to GATs.

Euh, what is a GAT?


Groeten
Geert Stappers
-- 
Silence is hard to parse

**Comment by stappersg** [comment]
FWIW
```patch
--- a/README.md
+++ b/README.md
@@ -40,4 +40,4 @@ Others
 ------
 
 Other implementations for USB in Rust
-* [embassy-usb](https://github.com/embassy-rs/embassy/blob/master/embassy-usb/src/driver.rs)
+* [embassy-usb](https://github.com/embassy-rs/embassy/blob/master/embassy-usb/src/driver.rs), an async variant.
```
is been _pushed_.

**Review by ryan-summers — APPROVED** [review]

**Comment by ryan-summers** [comment]
> Euh, what is a GAT? Groeten Geert Stappers -- Silence is hard to parse

[Generic Associated Types](https://github.com/rust-lang/rust/issues/44265) = GAT, and are not yet stabilized. They're generally used for defining async trait variants
